src/messenger-shortcuts.js

```diff
@@ -129,7 +129,13 @@ function focusMessageInput() {
 }
 
 function selectFirstSearchResult() {
-  var contacts = document.querySelectorAll('ul[role="listbox"]')[1];
+  var listboxes = document.querySelectorAll('ul[role="listbox"]');
+  // Checking if only a single character has been pressed
+  var contacts = listboxes[(getSearchBar().value.length() <= 1) ? 0 : 1];
+    contacts = document.querySelector('ul[role="listbox"]');
```

Comment:
Did you convert your if-else block into a ternary and then forget to delete your old code?